public/components/ui/model-edit-property/model-edit-property.stache

```diff
@@ -1,21 +1,19 @@
 <can-import from="./model-edit-property.less" />
 <can-import from="~/lib/stache-helpers/" />
+<can-import from="can-event-dom-enter/add-global/add-global"/>
```

Comment:
The `enter` event is no longer included by default.  This adds it to the global event registry.